packages/pg-pubsub/__tests__/subs.test.ts

```diff
@@ -23,7 +23,7 @@ const init = async (options: PostGraphileOptions = {}) => {
     throw new Error("CTX wasn't torn down");
   }
   const pgPool = new Pool({
-    connectionString: process.env.TEST_DATABASE_URL || "pggql_test",
```

Comment:
Given that all other packages require the `TEST_DATABASE_URL` to have a variable anyway, I thought it wouldn't be an issue to drop this fallback. (I also removed the "historic" name from the rest of the docs - sorry @benjie)